ts/session/sending/MessageQueue.ts

```diff
@@ -146,6 +146,30 @@ export class MessageQueue {
     await this.process(PubKey.cast(ourPubKey), message, sentCb);
   }
 
+  /**
+   * Sends a message that awaits until the message is completed sending
+   * @param user user pub key to send to
+   * @param message Message to be sent
+   */
+  public async sendToPubKeyNonDurably(
+    user: PubKey,
+    message: ClosedGroupNewMessage
+  ): Promise<boolean> {
+    let rawMessage;
+    let wrappedEnvelope;
```

Comment:
no need for the `wrappedEnvelope` to be a let, you can just have  a const in the `try`